worker-jobtracking/src/test/java/com/hpe/caf/worker/jobtracking/JobTrackingWorkerFactoryTest.java

```diff
@@ -26,18 +26,17 @@
 import com.hpe.caf.worker.tracking.report.TrackingReportStatus;
 import com.hpe.caf.worker.tracking.report.TrackingReportTask;
 import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mockito;
-import static org.mockito.Mockito.verify;
 import org.mockito.runners.MockitoJUnitRunner;
-
-import java.util.Collections;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.Map;
 import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.verify;
+import java.util.Date;
```

Comment:
I know you've just re-organised the imports here and the change it harmless, but it still takes time for me to look at it and realize that that is what's happened.  This is not necessary as part of the SCMOD-11919 changes.  There is no issue reorganizing the imports in another PR that is for that purpose and that would do it across the whole project if it is annoying to you.

Please take a look at these articles to help to understand where I am coming from:
https://github.com/salemove/github-review-helper/blob/master/doc/rules.md
https://mtlynch.io/code-review-love/

I do hate to keep banging on about the same thing.